handler/handler_test.go

```diff
@@ -60,12 +60,13 @@ var (
 )
 
 func TestContentPublishedHandler_Handle(t *testing.T) {
-	expectedSearchDataImport := marshalSearchDataImport(t, searchDataImportEvent)
 
 	kafkaProducerMock := &kafkatest.IProducerMock{
 		ChannelsFunc: getChannelFunc,
 	}
 
+	//for mock marshaller
+	expectedSearchDataImport := marshalSearchDataImport(t, expectedSearchDataImportEvent)
```

Comment:
It's worth having a test for when the data returned from Zebedee contains optional fields too:

1) All fields are returned from Zebedee
2) Only mandatory fields are returned from Zebedee

Reply:
tests updated for both scenarios

Reply:
tests updated to include both scenarios